logdna/common_test.go

```diff
@@ -0,0 +1,141 @@
+package logdna
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+const tmplPc = `provider "logdna" {
+	%s
+}`
+const tmplRs = `%s %q %q {
+%s
+}`
+
+var nilLst = []string{}
+var nilOpt = map[string]map[string]string{}
+
+var rsDfts = map[string]map[string]string{
+	"alert": {
+		"name": `"test"`,
+	},
+	"view": {
+		"apps":       "",
+		"categories": "",
+		"hosts":      "",
+		"levels":     "",
+		"name":       `"test"`,
+		"query":      `"test"`,
+		"tags":       "",
+	},
+}
+var chDfts = map[string]map[string]string{
```

Comment:
NIT: Summarizing the word `Defaults` is maybe going to far, effective go says that names should be:

> short, concise, evocative

`Dfts` is short but not evocative :)

Reply:
Yeah that's fair, I wasn't sure where to draw the line for concise vs descriptive/evocative variable names. Updated with more explicitly naming for `Defaults`, `Props`, and `Schema`